src/main/java/com/scriptbasic/syntax/commands/CommandAnalyzerMethod.java

```diff
@@ -1,16 +1,17 @@
 package com.scriptbasic.syntax.commands;
 
+import java.util.ArrayList;
```

Comment:
Please try not to rearrange the imports. It is a kind of noise in the pull request. I was searching 30sec why `ArrayList` was introduced to the code when it is only replacing string literals to constants before I realized that it is only the rearrangement of the imports.